spec/_attachments/ic.did

```diff
@@ -224,6 +231,21 @@ service ic : {
   bitcoin_send_transaction: (send_transaction_request) -> ();
   bitcoin_get_current_fee_percentiles: (get_current_fee_percentiles_request) -> (vec millisatoshi_per_byte);
 
+  // Canister backup and restore
+  take_snapshot: (record { 
```

Comment:
Would it make sense to call these "canister snapshots" to make it clear that it's snapshotting a canister? Maybe in the future we'll take snapshots of other things (e.g. the whole subnet, or a canister group). @mraszyk WDYT?

```suggestion
  take_canister_snapshot: (record { 
```

Reply:
I think so, I can add `canister` to all methods.

Reply:
Sounds good to me.